autoload/leaderf/python/leaderf/fileExpl.py

```diff
@@ -72,6 +87,7 @@ def _getFiles(self, dir):
         return file_list
 
     @showRelativePath
+    @showDevIcons
```

Comment:
`@showDevIcons` should be above the `@showRelativePath`.

Reply:
Thanks.
I fixed.